modules/simple_forms_api/spec/requests/v1/uploads_spec.rb

```diff
@@ -309,20 +309,27 @@
   end
 
   describe '#submit_supporting_documents' do
-    it 'renders the attachment as json' do
+    it 'renders the attachment as json for multiple form IDs' do
       clamscan = double(safe?: true)
       allow(Common::VirusScan).to receive(:scan).and_return(clamscan)
       file = fixture_file_upload('doctors-note.gif')
-      data = { form_id: '40-0247', file: }
 
-      expect do
-        post '/simple_forms_api/v1/simple_forms/submit_supporting_documents', params: data
-      end.to change(PersistentAttachment, :count).by(1)
+      # Define data for both form IDs
+      data_sets = [
+        { form_id: '10-10D', file: file },
+        { form_id: '40-0247', file: file }
+      ]
 
-      expect(response).to have_http_status(:ok)
-      resp = JSON.parse(response.body)
-      expect(resp['data']['attributes'].keys.sort).to eq(%w[confirmation_code name size])
-      expect(PersistentAttachment.last).to be_a(PersistentAttachments::MilitaryRecords)
+      data_sets.each do |data|
```

Comment:
I might rewrite this as something like:
```ruby
SUPPORTED_FORMS = ['1010-D', '40-0247']
SUPPORTED_FORMS.each do |form_id|
  data = { form_id:, file: }
  expect do
    ...
  end
  ...
end
```
It's a minor difference but I don't know what utility is gained from having a `data_sets` array, when the only thing that is changing is the form id.